bot/code_review_bot/tasks/clang_tidy_external.py

```diff
@@ -127,6 +130,13 @@ def as_markdown(self):
             ),
         )
 
+    def is_publishable(self):
```

Comment:
You don't need to override `is_publishable`, you can override `validates` instead (and this way you don't have to manually check `super().is_publishable`).

Reply:
And actually, with the modification below (in the call to the ExternalTidyIssue constructor), you shouldn't need to override this.
`validates` in ClangTidyIssue checks if publishable_check is True, and publishable_check is the value you pass with `publish` argument.

Reply:
gotcha. removing this bit then. thanks!

Reply:
@mozfreddyb this is still there :)